Makefile

```diff
@@ -26,7 +27,8 @@ drafter: config.gypi $(BUILD_DIR)/Makefile
 	cp -f $(BUILD_DIR)/out/$(BUILDTYPE)/$@ ./bin/$@
 
 install: drafter
-	cp -f $(BUILD_DIR)/out/$(BUILDTYPE)/drafter $(DESTDIR)/drafter
+	mkdir -p $(BINDIR)
+	cp -f $(BUILD_DIR)/out/$(BUILDTYPE)/drafter $(BINDIR)/drafter
```

Comment:
The average guy do not want to install the libraries and headers. Only the people who develop using drafter want them. And they probably don't install them to /usr/local rather installing them to their own customised places.